utils/defs.tex

```diff
@@ -399,6 +399,7 @@
 \vspace{-2em}
 }
 {
+\end{description} % ends \begin{description} from \apidescription
```

Comment:
This change appears to mess up the indenting of the "Notes to Implementors", e.g. in shmem_malloc and friends. Wondering if it would be more robust to have start and end macros for apidescription, since the last thing in the API description varies?

Reply:
This is easy to fix, but breaks one other case:

https://github.com/nspark/specification/blob/7e5cd8f9f17de95eafaec0250009c620f9c622dc/content/interoperability.tex#L37-L45

Other than that usage, `\apiimpnotes` is not used outside of a `description` environment anywhere in the Specification, despite the appearance in the Execution Model:

https://github.com/nspark/specification/blob/7e5cd8f9f17de95eafaec0250009c620f9c622dc/content/execution_model.tex#L44

I'm inclined (and have a WIP) to go with the easy fix (that corrects the styling errors you note) and "fix" the use of `\apiimpnotes` in the Interoperability section with something comparable to what's in the Execution Model section. What do you think? This would be _much less_ churn than making "SYNOPSIS" and "DESCRIPTION" proper environments (which would be a better overall solution, but would touch every API source file).

Reply:
Makes sense to me. IIUC, `\apiimpnotes` should only be used within an API description block. Would be good to introduce a different implementation note macro that can be used outside of API descriptions.

Reply:
:+1: I was working on that, actually, but stopped when I had some whitespace issues in the usage under "Execution Model". I'll pick that back up either this week or next.